### PR TITLE
change font to Lato; improve legibility of numbers on map

### PR DESCRIPTION
**Pull request by yzemaze**
Except for the logo margins (around .) and bold on game menu hover Lato worked out of the box for text, too.
Logo: Keep old one / use Lato, too?

### CODE REVIEW

File: public/assets/main.css [hunk 1]
```diff
@@ -1,11 +1,20 @@
...
+}
 
 a {
   color: currentColor;
   text-underline-offset: 0.2em;
 }
+nav a {
+  text-decoration: none;
+}
+nav a.active {
```
Comment (tobymao) [inline public/assets/main.css:13]: does this need to be a class? if it's not does the focus not work?

**Comment by tobymao** [comment]
everything lato

**Comment by tobymao** [comment]
Screen Shot 2020-07-05 at 8 16 55 AM

3 digit revenue too big

File: public/assets/main.css (hunk 1 above)
Comment (yzemaze) [reply public/assets/main.css:13]: No, it was just the/my usual way. `route_anchor == anchor[1..-1] ? a_props[:style][:textDecoration] = 'underline' : ''` would work, too.

Comment (tobymao) [reply public/assets/main.css:13]: ok let’s revert it

**Comment by yzemaze** [comment]
> everything lato

?
Is it not?

> 3 digit revenue too big

I know. Do you
a) want it to fit the small circle
b) the circle to expand
or c) neglect this edge case?

**Comment by tobymao** [comment]
ideally fit in the circle

**Pull request merged**